central/complianceoperator/v2/scanconfigurations/service/convert.go

```diff
@@ -248,6 +248,68 @@ func getLatestBindingError(status *storage.ComplianceOperatorStatus) string {
 	return ""
 }
 
+func convertStorageReportDataToV2ScanStatus(ctx context.Context, reportData *storage.ComplianceOperatorReportData, notifierDS notifierDS.DataStore) (*v2.ComplianceScanConfigurationStatus, error) {
+	if reportData == nil {
+		return nil, nil
+	}
+	notifiers := make([]*v2.NotifierConfiguration, 0, len(reportData.GetScanConfiguration().GetNotifiers()))
+	for _, notifierConfig := range reportData.GetScanConfiguration().GetNotifiers() {
+		notifier, found, err := notifierDS.GetNotifier(ctx, notifierConfig.GetId())
```

Comment:
nit:  A comment for why we have to get the notifier first to then get the name would be beneficial.

Reply:
Done in [37708f8](https://github.com/stackrox/stackrox/pull/13804/commits/37708f8edb6c5dca6d40463f28dc981edceeec49)